src/auto-complete.jsx

```diff
@@ -341,7 +341,15 @@ const AutoComplete = React.createClass({
       switch (typeof item) {
         case 'string':
           if (this.props.filter(this.state.searchText, item, item)) {
-            requestsList.push(item);
+            requestsList.push({
+              text: item,
+              value: (
+                <MenuItem
+                  innerDivStyle={{overflow: 'hidden'}}
+                  value={item}
+                  primaryText={item}
+                />)
```

Comment:
Add a comma:` />),`, and possibly put `),` on its own line. (Not sure what the preferred house style is?)